homework/index.js

```diff
@@ -30,14 +36,102 @@
     return elem;
   }
 
+  function createOptions(repoNames, select) {
+    createAndAppend('option', select, { text: 'Select module', disabled: 'disabled' });
+    repoNames.sort((a, b) => a.name.localeCompare(b.name));
+    repoNames.forEach((repository, i) => {
+      createAndAppend('option', select, {
+        text: repository.name,
+        value: repository.name,
+      });
+    });
+  }
+
+  function renderRepoInformation(repo, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repo}`;
+    fetchJSON(url, (err, repoInfos) => {
+      if (err) {
+        renderError(err);
+        return;
+      }
+
+      ul.innerHTML = ' ';
+      const li = createAndAppend('li', ul);
+      const table = createAndAppend('table', li, { class: 'infoTable' });
+      const tbody = createAndAppend('tbody', table);
+      const repoName = createAndAppend('tr', tbody);
+      createAndAppend('td', repoName, { text: 'Repository:', class: 'explanations' });
+      const repoNameTd = createAndAppend('td', repoName, {
+        class: 'list-item',
+      });
+      createAndAppend('a', repoNameTd, {
+        href: `${repoInfos.html_url}`,
+        target: 'blank',
+        text: `${repoInfos.name}`,
+      });
+      const description = createAndAppend('tr', tbody);
+      createAndAppend('td', description, { text: 'Description:', class: 'explanations' });
+      createAndAppend('td', description, { text: `${repoInfos.description}` });
+      const fork = createAndAppend('tr', tbody);
+      createAndAppend('td', fork, { text: 'Forks:', class: 'explanations' });
+      createAndAppend('td', fork, { text: `${repoInfos.forks}` });
+      const updateTime = createAndAppend('tr', tbody);
+      createAndAppend('td', updateTime, { text: 'Updated:', class: 'explanations' });
+      createAndAppend('td', updateTime, { text: `${repoInfos.updated_at}` });
+    });
+  }
+
+  function renderContributorsInformation(subjects, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${subjects}/contributors`;
+    fetchJSON(url, (error, contributorDetails) => {
+      if (error) {
+        renderError(error);
+        return;
+      }
+
+      ul.innerHTML = ' ';
+      createAndAppend('h4', ul, { text: 'Contributors' });
+      contributorDetails.forEach(contributor => {
+        const li = createAndAppend('li', ul);
```

Comment:
It would be nice if you could click anywhere in the list item (not just on the login name) to go to the GtiHub page of the contributor. You can achieve that by wrapping the complete inner content of the `li` into an `a` tag with the correct `href` value. You then should not use an `a` tag anymore for the login name, but just a div or a span.